lib/systemd_testsuite_test.pm

```diff
@@ -20,17 +20,26 @@ use known_bugs;
 use testapi;
 use power_action_utils 'power_action';
 use utils 'zypper_call';
-use version_utils;
+use version_utils qw(is_opensuse is_sle is_tumbleweed);
```

Comment:
Why make it so complicated?
`version_utils` should by default export that functions.

Reply:
This is a declaration of used functions.
When those functions are not used, we can remote the `use version utils`.
I have sometimes found unused "use" sentences. And it is a nightmare to check for yet used functions.